config/foreman_maintain.yml.packaging

```diff
@@ -0,0 +1,16 @@
+# Directory where the logs are stored.
+# The default file is named as /log/foreman_maintain.log
+:log_dir: '/var/log/foreman-maintain/'
+
+# Logger levels: mention one of debug, info, warning, error, fatal
+:log_level: 'error'
```

Comment:
Or perhaps just  leave it commented out so that we can control the default log level from the code, and set when it's ok to switch from debug to error by default.